src/systemdhook.c

```diff
@@ -790,15 +790,6 @@ int main(int argc, char *argv[])
 		return EXIT_FAILURE;
 	}
 
-	/* Extract values from the state json */
```

Comment:
This field no longer exists in the state.json file, which is why it was moved.  Older versions of runc provided the value in the state json, newer only in the config.json.